chromadb/test/test_api.py

```diff
@@ -6,15 +6,19 @@
 import pytest
 import tempfile
 import numpy as np
+import os
+import shutil
 from datetime import datetime, timedelta
 from chromadb.utils.embedding_functions import (
     DefaultEmbeddingFunction,
 )
 
+persist_dir = tempfile.mkdtemp()
```

Comment:
mkdtemp seems like an improvement over gettempdir

total nit, but would it make sense to use https://docs.python.org/3.4/library/tempfile.html#tempfile.TemporaryDirectory and let it clean up itself? this is mostly for my learning

Reply:
It's mostly because its useful for debugging to remove the deletion and check the contents of the dir